uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/CheatSheetsTest.java

```diff
@@ -0,0 +1,146 @@
+			+ "in the menu bar at the top of the workbench. This step changes the perspective "
+			+ "to set up the Eclipse workbench for development of your Apache Camel/Red Hat Fuse " + "routes.";
+
+	public static final String WIZARD = "To create your new project you now need to "
+			+ "open the New Integration Project wizard. You can do that by selecting "
+			+ "File > New > Fuse Integration Project from the top menu bar.Once it "
+			+ "opened, you can specify the location your project should live in. On "
+			+ "the next page, you can select specify the target environment for deploying "
+			+ "your new project and choose a specific Camel version. On the last page, you "
+			+ "can select from a list of available templates. Please select the template "
+			+ "Simple log using Spring Boot - Spring DSL from the list.Click on Finish to "
+			+ "finally create your new project.";
+
+	@AfterClass
+	public static void cleanWorkspace() {
+		new CleanWorkspaceRequirement().fulfill();
+	}
+
+	/**
+	 * <p>
+	 * Fuse Integration Project cheat sheet test
+	 * </p>
+	 * <b>Steps</b>
+	 * <ol>
+	 * <li>Open Help -> Cheat Sheets -> Red Hat Fuse -> Create a Fuse integration project</li>
+	 * <li>Check if texts in all sections are actual.</li>
+	 * <li>Activate available HyperLinks and wait for action.</li>
+	 * <li>Check according actions.</li>
+	 * <li>Go through New Project Wizard.</li>
+	 * </ol>
+	 */
+	@Test
+	public void testCheatSheets() {
+
+		/*
+		 * For JRE8 check.
+		 */
+		boolean hasJava8 = JDK8Check.isJava8Available();
+
+		/*
+		 * Open Help -> Cheat Sheets -> Red Hat Fuse -> Create a Fuse...
+		 */
+		CheatSheetsWizard wizard = new CheatSheetsWizard();
+		wizard.open();
+		wizard.selectCheatSheet("Red Hat Fuse", "Create a Fuse Integration Project based on Camel");
+		wizard.finish();
+
+		/*
+		 * Open new Workbench View for Cheat Sheets.
+		 */
+		CheatSheetsView cheatsheets = new CheatSheetsView();
+		cheatsheets.open();
+
+		/*
+		 * Introduction part After activating 'Click to Begin' introductionLink text changes to 'Click to Restart'.
+		 */
+		assertTrue("Introduction text not matching.", cheatsheets.sectionHasText("Introduction", INTRODUCTION));
+		cheatsheets.selectHyperlink("Click to Begin");
+		assertTrue("Introduction hyperlink was not fulfilled correctly. Link text should be changed to Click to Restart.",
+				cheatsheets.hyperlinkTextChange("Click to Restart"));
+
+		/*
+		 * Open the Fuse Integration Perspective part After activating 'Click to perform' perspectiveLink text changes
+		 * to 'Click to redo'.
+		 */
+		assertTrue("Perspective text not matching.",
+				cheatsheets.sectionHasText("Open the Fuse Integration Perspective", PERSPECTIVE));
+		cheatsheets.selectHyperlink("Click to perform");
+		assertTrue("Perspective hyperlink was not fulfilled correctly. Link text should be changed to Click to redo.",
+				cheatsheets.hyperlinkTextChange("Click to redo"));
```

Comment:
indendation issue

Reply:
@apupier Actually it's just formatting of long line, not two commands.

Reply:
oups, I read too fast. Sorry